testing/sdk_cmd.py

```diff
@@ -79,7 +81,8 @@ def cluster_request(
     url = shakedown.dcos_url_path(cluster_path)
     # consistently include slash prefix for clearer logging below
     cluster_path = "/" + cluster_path.lstrip("/")
-    log.info("(HTTP {}) {}".format(method.upper(), cluster_path))
+    if log_output:
```

Comment:
An alternative would be to have a the global `log` object be a wrapper for the current `log` object and then enable or disable the logging in that class instead of having to add it to every line here?